src/main/java/mate/controller/car/GetAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet("/cars")
+public class GetAllCarsController extends HttpServlet {
```

Comment:
```suggestion
public class DisplayAllCarsController extends HttpServlet {
```

Reply:
Here I am just getting all the cars and passing them to `file.jsp`. I think `Get` is better here